autobahn/wamp_arguments.hpp

```diff
@@ -45,6 +45,78 @@ using wamp_kw_arguments = std::unordered_map<std::string, msgpack::object>;
 static const msgpack::object EMPTY_ARGUMENTS(std::array<msgpack::object, 0>(), nullptr);
 static const msgpack::object EMPTY_KW_ARGUMENTS(wamp_kw_arguments(), nullptr);
 
+
+//msgpack map utilities.  
+//TODO: refactor event & invocation to used these
+template <typename T>
+inline T value_for_key(const msgpack::object& object, const std::string& key)
+{
+	if (object.type != msgpack::type::MAP) {
```

Comment:
Based on the size of indentations that the web interface is showing, it appears that this commit uses tabs to indent rather than spaces. This is inconsistent with the code so far and might lead to weird-looking code in some people's editors.